scripts/ci/windows_flake_rate_proof.py

```
@@ -0,0 +1,447 @@
+    measurement's load behind and poison every later iteration.
+    """
+    if sys.platform == "win32":
+        run(["taskkill", "/T", "/F", "/PID", str(pid)], timeout=60)
+    else:
+        run(["pkill", "-KILL", "-P", str(pid)], timeout=60)
+        run(["kill", "-KILL", str(pid)], timeout=60)
+
+
+def run_with_ceiling(command: list[str], ceiling_s: float) -> tuple[int | None, str, bool]:
+    """Return (exit code or None when timed out, combined output, timed_out)."""
+    process = subprocess.Popen(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)